### DISCUSSION TITLE
Rootless Xorg on Debian

### DISCUSSION THREAD

**Discussion by fingolfon** [lxqt/lxqt#2435] [Q&A]
Hi guys! Trying out some of the spins of Fedora 38 I noticed both KDE and LXQT are running Xorg as user, not as root.  That's not the case on Lubuntu or Debian LXQT.  So I was wondering what's the way to configure it?

**Comment by tsujan** [comment]
It may depend on the display manager, not the distro. See https://wiki.archlinux.org/title/xorg#Rootless_Xorg

It seems that SDDM also supports it, but I didn't dig into it.

**Reply by yan12125** [reply]
SDDM git-master supports rootless Xorg, but with paper cuts.

**Reply by tsujan** [reply]
> SDDM git-master supports rootless Xorg, but with paper cuts.

Strange that ArchWiki is silent about it. There are pieces of info in other places, but I'm not sure they're reliable.
